pkgs/top-level/all-packages.nix

```diff
@@ -12348,16 +12348,19 @@ with pkgs;
     fftw = fftwSinglePrec;
   };
 
-  inherit
-    ({
-      timeshift-unwrapped = callPackage ../applications/backup/timeshift/unwrapped.nix { };
-      timeshift = callPackage ../applications/backup/timeshift { grubPackage = grub2; };
-      timeshift-minimal = callPackage ../applications/backup/timeshift/minimal.nix { };
-    })
-    timeshift-unwrapped
-    timeshift
-    timeshift-minimal
-    ;
+  timeshift-minimal = callPackage ../by-name/ti/timeshift/package.nix { };
```

Comment:
Since you are calling the other package the wrapped version, this is the unwrapped version.  We are not changing anything about the program and if all the tools are in the system anyway, they behave the same.